recipes/kitty/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "kitty" %}
+{% set version = "0.14.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/kovidgoyal/{{ name }}/releases/download/v{{ version }}/{{ name }}-{{ version }}.tar.xz
+  sha256: 9eb32808425867dd63580718bc1b97bbd9046d4024279837c6132eb1ef37f189
+
+build:
+  number: 0
+  script: |
+    export PKG_CONFIG_PATH="${PREFIX}/lib/pkgconfig:/usr/lib64/pkgconfig:/usr/lib/x86_64-linux-gnu/pkgconfig:${PKG_CONFIG_PATH}"
+    export CFLAGS="${CFLAGS} -I ${PREFIX}/include/freetype2 -I ${PREFIX}/include/harfbuzz"
+    {{ PYTHON }} setup.py linux-package --prefix=$PREFIX --update-check-interval=0
```

Comment:
While I'm a fan of multu-line scripts this is marginally too complex to deserve its own `build.sh`.